userguide/getting-started/quick_start_with_kaui.adoc

```diff
@@ -0,0 +1,189 @@
+= Quick Start with Kaui
+:imagesdir: C:\_My Documents\FlowWritingLLC\Projects\Kill Bill\Documentation\killbill-docs\userguide\assets\img\getting-started
+
+Kaui is the back-office user interface for Kill Bill. This guide walks you through several tasks in Kaui. You will become familiar with moving around the application, and you'll learn some basic Kill Bill concepts:  
+
+. <<step1, Create a tenant>>
+. <<step2, Set up a simple catalog and plans>>
+. <<step3, Create an account>> 
+. <<step4, Add a payment method to the account>>
+. <<step5, Set up a subscription for the account>>
+. <<step6, View the generated invoice>>
+
+[NOTE]
+*Note:* You can also achieve these same results of this tutorial with the API. In fact, if you are going to have a front-end interface (such as a website), the API calls are what you use to communicate with Kill Bill (as opposed to using Kaui). For more information, see https://docs.killbill.io/latest/getting_started.html#_api_client[API Client] in the https://docs.killbill.io/latest/getting_started.html.html[_Getting Started_] guide.
+
+== Before You Begin
+
+Make sure you have installed Kill Bill and Kaui and launched them per the https://docs.killbill.io/latest/getting_started.html.html[_Getting Started_] guide. 
+
+== Additional Resources
+
+* https://docs.killbill.io/latest/userguide_kaui.html[_Kaui Guide_] 
+
+* https://killbill.github.io/slate[_API Reference_]
+
+* https://www.youtube.com/c/KillbillIoOSS[Kaui tutorials on YouTube]
+
```

Comment:
I would suggest either adding a link here to the [Kill Bill Glossary](https://docs.killbill.io/latest/Kill-Bill-Glossary.html) or linking individual terms (like tenant, catalog, etc.) to their definitions in the glossary.

Reply:
I added glossary links for the first instance of each term.

Reply:
FYI, I'm using the ^ character in the link syntax to see if it works in HTML to open the glossary term in a new browser tab. I have already observed that there's a problem with using it with PDF, which is a minor issue for us since we don't normally release in PDF.